README.txt

```diff
@@ -1,21 +1,24 @@
-Methodocracy
+Methodocracy (noun):
+A cultural-political ideology which proposes that optimal decision-making, from the individual to national level, is necessarily based upon the quantity and quality of data, interpretation, and conclusions gathered through tested and reliable methodologies.
```

Comment:
You are affiliated with Methodocracy Foundation now so please don't, someone else will.